.istanbul.yml

```diff
@@ -0,0 +1,6 @@
+check:
+  global:
+    statements: 80
+    lines: 80
+    branches: 37
```

Comment:
Due to the minimal nature of the code, the branch coverage is 37.5%. This will increase as the project gets fleshed out. I'm adding another test to enable functions to be set to 80%